platform/commonUI/general/src/directives/MCTSplitPane.js

```diff
@@ -134,6 +132,11 @@ define(
 
                     // Get actual size (to obey min-width etc.)
                     firstSize = getSize(first[0]);
+
+                    if (alias === 'mctSplitPane-imagery') {
+                        firstSize = firstSize - 5;
```

Comment:
@dtailor90 I'm trying to figure out why this is hardcoded here.  Reason?

Reply:
Let's get rid of this and change CSS so that we can consistently read size without this workaround here.

Reply:
@larkin - removed the workaround from MCTSplitPane and added box sizing to the thumbnail imagery wrapper.